src/cloud-cli/meltano/cloud/api/client.py

```diff
@@ -232,18 +232,28 @@ async def run_project(
         Args:
             deployment: The name of the Meltano Cloud deployment in which to run.
             job_or_schedule: The name of the job or schedule to run.
+
+        Raises:
+            MeltanoCloudError: The Meltano Cloud API responded with an error.
         """
         async with self.authenticated():
-            url = (
-                "/run/v1/external/"
-                f"{self.config.tenant_resource_key}/"
-                f"{self.config.internal_project_id}/"
-                f"{deployment}/{job_or_schedule}"
-            )
-            await self._json_request(
-                "POST",
-                url,
-            )
+            try:
+                url = (
+                    "/run/v1/external/"
+                    f"{self.config.tenant_resource_key}/"
+                    f"{self.config.internal_project_id}/"
+                    f"{deployment}/{job_or_schedule}"
+                )
+                await self._json_request(
+                    "POST",
+                    url,
+                )
+            except MeltanoCloudError as ex:
+                ex.response.reason = (
+                    f"Unable to find schedule with name {job_or_schedule!r} "
```

Comment:
```suggestion
                    f"Unable to find schedule named {job_or_schedule!r} "
```

Nitpick - I know this error message is the same as the one from other parts of the Cloud CLI, but I've been meaning to update those too.